packages/block-library/src/group/index.js

```diff
@@ -25,6 +26,27 @@ export const settings = {
 		anchor: true,
 		html: false,
 	},
+
+	transforms: {
```

Comment:
I like that you were able to plug this new logic into the existing `transforms` logic 👍